File: AI_Lab/lab4/lab3/task1.py
sentence = input("Enter a sentence: ")
book={}

word = sentence.lower().split()
count = 0
for i in word :
    for j in word:
       if (i == j):
           count = count + 1   
    book[i] = count

    count = 0  

book.pop("the",None)
book.pop("a", None)
book.pop("is", None)
book.pop("am", None)
book.pop("and", None)
print(book)

File: shayyanbhaichallange/task1.cpp
#include <iostream>
using namespace std;

int main()
{
    string choice;
    cout << "enter";
    cin >> choice;
    int count = choice.length() -2;
  
   
    cout << choice[0];
    cout << count;
    cout << choice[count +1];
    return 0;
}
